beacon-chain/rpc/eth/beacon/blocks.go

```diff
@@ -813,6 +813,77 @@ func (bs *Server) GetBlindedBlock(ctx context.Context, req *ethpbv1.BlockRequest
 		return nil, status.Errorf(codes.Internal, "Could not get signed beacon block: %v", err)
 	}
 
+	capellaBlk, err := blk.PbCapellaBlock()
```

Comment:
I refactored the code and put it into its own file. Deduplicating is hard because there are lot of differences in terms of block types and function calls.